src/components/Base/Popover/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Tooltip from '../Tooltip';
+import styles from './index.scss';
+
+export default class Popover extends React.Component {
+  static propTypes = {
+    prefixCls: PropTypes.string,
+    className: PropTypes.string,
+    trigger: PropTypes.string,
+    placement: PropTypes.string,
+    title: PropTypes.string,
+    content: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.element,
+    ]),
+    disabled: PropTypes.bool,
+    visible: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    prefixCls: 'pi-popover',
+    trigger: 'click',
+    placement: 'bottom',
+    disabled: false,
+  };
+
+  getOverlay = () => {
+    const { prefixCls, title, content } = this.props;
+
+    return (
+      <div>
+        {title && <div className={`${prefixCls}-title`}>{title}</div>}
```

Comment:
if `prefixCls` is empty string, `${prefixCls}-title` is `-title`, is it right ?

Reply:
prefixCls will use defaultProps value if no value parsed.

Reply:
i know, i mean, defaultProp of `prefixCls` could be `pi-popover-`, `${prefixCls}-title` change as `${prefixCls}title` is better, so when `prefixCls` is `''`, that concat string will not prefixed with a `-`

Reply:
If put `-`, it does not look like `prefixCls`.
When we set this property, `-` is usually not added.